packages/react-union/src/components/Union/Union.js

```diff
@@ -19,83 +22,149 @@ class Union extends Component {
 		 */
 		children: PropTypes.node,
 		/**
-		 * Called after the scan of the HTML is done.
+		 * Initial props retrieved by the SSR server. Passed by `withServerContext`.
 		 */
-		onScanEnd: PropTypes.func,
+		initialProps: PropTypes.arrayOf(PropTypes.object),
 		/**
-		 *  Called when there is an error while scanning of the HTML.
+		 * Whether the component is rendered in SSR context. Passed by `withServerContext`.
+		 */
+		isServer: PropTypes.bool.isRequired,
+		/**
+		 * Called when there is an error while scanning of the HTML.
 		 */
 		onScanError: PropTypes.func,
 		/**
-		 * Called before the scan of the HTML
+		 * Called before the scan of the HTML.
 		 */
 		onScanStart: PropTypes.func,
 		/**
-		 * Element in which the scan is running. By default `document`.
+		 * Called after the scan of the HTML is successfully done.
+		 */
+		onScanSuccess: PropTypes.func,
```

Comment:
This is one of the public API changes in react-union itself. I thought that the name `onScanSuccess` goes better with `onScanError`, and since I've modified the signature ever so slightly, I thought I should go for the rename as well. Let me know if I should rename it back, we can have this discussion later if you want.